modules/StoreClass.js

```diff
@@ -0,0 +1,37 @@
+class Store {
+  // Counter to keep track of books index
+  constructor() {
+    this.count = this.getBooks().length + 1;
+  }
+
+  // Call to books from storage with get item
+  getBooks() {
+    if (localStorage.getItem('books') === null) {
+      this.books = [];
+    } else {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
+    return this.books;
+  }
+
+  addBook(book) {
+    const newBook = {
+      id: this.count,
+      title: book.title,
+      author: book.author,
+    };
+
+    const books = this.getBooks();
+    books.push(newBook);
+    localStorage.setItem('books', JSON.stringify(books));
+    this.count += 1;
+  }
```

Comment:
- [OPTIONAL] It would be a good UI design to display a message that tells users if books have been added when they click the _**Add**_ button. 👍